hera_cal/lst_stack/averaging.py

```diff
@@ -681,11 +735,15 @@ def average_and_inpaint_simultaneously(
             stackd = stack.data[:, iap, :, polidx]
             stackf = complete_flags[:, iap, :, polidx]
             stackn = np.abs(stack.nsamples[:, iap, :, polidx])
-            flagged_mean = lstavg['data'][iap, :, polidx]
-            avg_flgs = lstavg['flags'][iap, :, polidx]
+            flagged_mean = lstavg["data"][iap, :, polidx]
+            avg_flgs = lstavg["flags"][iap, :, polidx]
 
+            pol1, pol2 = utils.split_pol(pol)
```

Comment:
same note as before, I'd call these `jpol1` or `antpol1` or `ap1` or `ant_1_pol` or something like that.